### ISSUE TITLE
Website not responsive for Screens.

### ISSUE THREAD

**Issue by gitvivek14** [anAcc22/another_graph_editor#2]
Horizontal Scroll , Vertical Scroll , not properly fixed.
not ui friendly.
![image](https://github.com/user-attachments/assets/9c83007c-a3c7-4074-8a47-34719e9bb591)

**Comment by gitvivek14** [comment]
i can fix this , i you allow to me as a contributor.

**Comment by anAcc22** [comment]
Hi, can you fork the repo and make a PR?

**Comment by gitvivek14** [comment]
sure , i will do it.

**Comment by gitvivek14** [comment]
> sure , i will do it.


also can i get some recognition after the work is done.

**Comment by anAcc22** [comment]
Yup sure, I think you'll be shown as a contributor on the right panel (e.g., [this](https://github.com/nvim-neorg/neorg)). I'm not too familiar with Github so if that doesn't work, I could instead create a table/list in the README showing all contributions. Would that be okay with you?

**Comment by anAcc22** [comment]
Hey man, I've set overflow to hidden for all textareas except Current Nodes, because someone on Codeforces brought it up and I decided to remove the scrollbars to make it easier to read. Was that the UI issue you were referring to? Or was it something else?

**Comment by gitvivek14** [comment]
> Yup sure, I think you'll be shown as a contributor on the right panel (e.g., [this](https://github.com/nvim-neorg/neorg)). I'm not too familiar with Github so if that doesn't work, I could instead create a table/list in the README showing all contributions. Would that be okay with you?

this would be fine.

**Comment by gitvivek14** [comment]
> Hey man, I've set display to none for the scrollbars of all textareas because someone on Codeforces brought it up and I decided to remove the scrollbars to make it easier to read. Was that the UI issue you were referring to? Or was it something else?

it was something else

**Comment by anAcc22** [comment]
Hi, could you elaborate on the issue in more detail? What is the expected behavior/appearance, and how can I reproduce it? Thanks!
